proposals/lip-0070.md

```diff
@@ -434,6 +434,28 @@ def getClaimableRewards(stakerAddress: Address) -> dict[TokenID, uint64]:
     return rewards
 ```
 
+#### getExpectedReward
+
+This function calculates the expected reward for staking with a particular validator. 
+
+##### Parameters
+
+* `validatorAddress`: The address of the validator considering to stake with.
+* `validatorReward`: The hypothetical reward for the validator. 
```

Comment:
How is this reward supposed to be obtained? LIP 71 does not provide an endpoint at the moment.
Ideally, something like the function [getValidatorBlockReward](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0071.md#getvalidatorblockreward) would also be accessible. Then the front-end could easily compute the expected ROI or expected shared rewards per month/year.

Reply:
Maybe you could add an appendix then how to compute the ROI or rewards per month/year with these endpoints (in particular, based on the round length and block time).

Reply:
I assume that LIP 71 provides such an endpoint and the UI would have access to both endpoints (PoS and Dynamic Rewards).

Reply:
See also changes in LIP 71